.github/workflows/seahorn-main.yml

```diff
@@ -0,0 +1,52 @@
+# workflow to build and test seahorn main branch every night.
+# This version of seahorn will NOT be pushed to docker.
+
+name: Nightly-main-nopush
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: 0 0 * * *  # run every day at UTC 00:00
+  workflow_dispatch:
+    branches: main
+    inputs:
+      TriggerReason:
+        description: 'Triggered for reason:'
+        required: true
+        default: ''
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "test"
+  test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Log manual trigger info
+        if: ${{ github.event_name == 'workflow_dispatch' }}  # only if triggered manually
+        run: |
+          echo "Reason for manual trigger: ${{ github.event.inputs.TriggerReason }}"
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+        with:
+          ref: dev14  # only checkout dev14
```

Comment:
Not sure what I does but shouldn't be `main` ?

Reply:
I think @caballa is right. We want to test `main`. `dev14` is already tested. However, recently someone complained that `main` wasn't building -- hence this change. I'll update the branch.